### DISCUSSION TITLE
How to set a default shadow size for all Comboboxes?

### DISCUSSION THREAD

**Discussion by artemik** [mantinedev/mantine#6664] [Q&A] [answered]
I know you can set a shadow on a Combobox (or rather a Select) like this:
```
<Select comboboxProps={{ shadow: 'md' }} ... />
```
and the theme object allows overriding what shadow size variables mean.

But how can I set a default shadow size value for all comboboxes (or even all dropdowns)? The default Mantine shadow is zero as it seems, which makes dropdowns blend in with background UI. Theme object has "defaultRadius", but not something like "defaultShadow". Thanks.

**Comment by rtivital** [comment, marked as answer]
You can do that with default props – https://mantine.dev/theming/default-props/

**Reply by artemik** [reply]
That's even better, thanks! @rtivital
